tests/endpoint_decorators_test.py

```diff
@@ -42,3 +42,38 @@ def test_log_headers(*args, **kwargs):
                 "Authorization: REDACTED\n"
                 "authorization: REDACTED\n"
                 "SafeStuff: This better be logged this way")
+
+    def test_logbodies_errors_on_invalid_json(self):
+        """
+        """
+        data = "{\"invalid\": ...}"
+        valueErrorStr = "No JSON object could be decoded"
+        logger_mock = Mock()
+        logger_mock.info = Mock()
+        logger_mock.exception = Mock()
+        request_mock = type("FakeRequest", (), {
+            "headers": {
+                "content-type": "application/json"
+            },
+            "data": data
+        })
+        request_mock.get_data = Mock(return_value=data)
+        mock_container = type("FakeContainer", (), {
+            "logger": logger_mock,
+            "request": request_mock
+        })
+
+        @decorators.logbodies
+        def test_log_bodies(*args, **kwargs):
+            return request_mock
```

Comment:
Can you do this like this? No need for args and kwargs, also lets return the container request since iit is passed to us
```
@decorators.logbodies
def test_log_bodies(container):
    return container.request
```